### PR TITLE
lj/pj/aj/wj (and variants) can modify t1 in place

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: docs/ref/pj.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ x pj y     pj[x;y]
 
 Where
 
--   `x` and `y` are tables
+-   `x` and `y` are tables. Since 4.1t 2023.08.04 if `x` is the name of a table, it is updated in place.
```
Comment (JosieMD) [inline docs/ref/pj.md:20]: What about if y is the name of a table?

File: docs/ref/wj.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ wj1[w; c; t; (q; (f0;c0); (f1;c1))]
 
 Where
 
--   `t` and `q` are simple tables to be joined (`q` should be sorted `` `sym`time `` with `` `p# `` on sym)
+-   `t` and `q` are simple tables to be joined (`q` should be sorted `` `sym`time `` with `` `p# `` on sym). Since 4.1t 2023.08.04 if `t` is the name of a table, it is updated in place.
```
Comment (JosieMD) [inline docs/ref/wj.md:21]: And table q?

Comment (sshanks-kx) [reply docs/ref/wj.md:21]: would give an error, as would trying to feed in anything else for 'q' that isnt a table

Comment (JosieMD) [reply docs/ref/wj.md:21]: OK.

**Pull request merged**